README.md

```diff
@@ -1,6 +1,69 @@
+
+## Overview
+
+The [External Health Monitor](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1432-volume-health-monitor) is impelmeted as two components: `External Health Monitor Controller` and `External Health Monitor Agent`.
+
+- External Health Monitor Controller:
+  - The external health monitor controller will be deployed as a sidecar together with the CSI controller driver, similar to how the external-provisioner sidecar is deployed.
+  - Trigger controller RPC to check the health condition of the CSI volumes.
+  - The external controller sidecar will also watch for node failure events. This component can be enabled via a flag.
+
+- External Health Monitor Agent:
+  - The external health monitor agent will be deployed as a sidecar together with the CSI node driver on every Kubernetes worker node.
+  - Trigger node RPC to check volume's mounting conditions.
+  - Note that currently we do not have CSI support for local storage. When the support is available, we will implement relavant CSI monitoring interfaces as well.
+
+The External Health Monitor needs to invoke the following CSI interface to support
+
+- External Health Monitor Controller:
+  - NodeGetVolumeStats
+  - ControllerGetVolume
+- External Health Monitor Agent:
+  - ListVolumes
+
+## Usage
+
+Because External Health Monitor needs controller and agent simultaneously, you have to deploy them to your Kubernetes cluster.
+
+### External Health Monitor Controller
+
+```bash
+cd external-health-monitor
+kubectl create -f deploy/kubernetes/external-health-monitor-controller
+```
+
+### External Health Monitor Agent
+
+```bash
+kubectl create -f deploy/kubernetes/external-health-monitor-agent
+```
+
+You can run `kubectl get pods` command to confirm if they are deployed on your cluster successfully.
+
+```bash
+> kubectl get pods -o wide
+
+// agent
+
+csi-external-health-monitor-agent-flf4h                   2/2     Running   0          3m36s   172.18.4.6    xuran-testimported-w1         <none>           <none>
+csi-external-health-monitor-agent-l68vh                   2/2     Running   0          3m36s   172.18.5.7    imported2001-xuran-jp2v-dev   <none>           <none>
+csi-external-health-monitor-agent-r5w9r                   2/2     Running   0          3m36s   172.18.3.33   xuran-testimported-w2         <none>           <none>
+
+// controller
+
+csi-external-health-monitor-controller-85b5fc7c57-clsdk   2/2     Running   0          4m12s   172.18.4.5    xuran-testimported-w1         <none>           <none>
+csi-external-health-monitor-controller-85b5fc7c57-cw4lx   2/2     Running   0          4m12s   172.18.5.6    imported2001-xuran-jp2v-dev   <none>           <none>
+csi-external-health-monitor-controller-85b5fc7c57-d77xx   2/2     Running   0          4m12s   172.18.3.32   xuran-testimported-w2         <none>           <none>
+
+```
+After that, you can run both of commands as below to confirm if External Health Monitor works fine
```

Comment:
"After that, you can run both of commands as below to confirm if External Health Monitor works fine" -> "Check logs of external health monitor controller and agent as follows:"